pkg/resources/sources/cpu.go

```diff
@@ -31,13 +37,16 @@ func (c *CPU) GetTotalResources(ctx context.Context) (core.ResourceList, error)
 		hostCPUSum += int64(v.Cores)
 	}
 
-	resources := core.ResourceList{core.ResourceCPU: *resource.NewScaledQuantity(hostCPUSum, resource.Kilo)}
+	cpuQuantity := float64(hostCPUSum) * c.OvercommitVCPU
+	resources := core.ResourceList{
+		core.ResourceCPU: *resource.NewQuantity(int64(cpuQuantity*1000), resource.DecimalSI),
+	}
 
 	return resources, nil
 }
 
-func (c *CPU) GetName() string {
```

Comment:
why are you mixing source name with resource name?

Reply:
ok, I will create separate constants for the source names.

Reply:
However, we already have the source name associated with the resource name: [memory.go](https://github.com/ironcore-dev/libvirt-provider/blob/b1d44ebf879bfb8ea48a54a84198d8e827376e19/pkg/resources/sources/memory.go#L17).

Reply:
no, it is resource name for new resource, but in GetName() i returned hugepages strings

Reply:
Got it. I'll create a separate constant for the source name and use that instead of hardcoding strings in multiple places to ensure it can be managed from a single location.